spec/message.abnf

```diff
@@ -104,5 +104,5 @@ quoted-escape   = backslash ( backslash / "|" )
 reserved-escape = backslash ( backslash / "{" / "|" / "}" )
 backslash       = %x5C ; U+005C REVERSE SOLIDUS "\"
 
-; Whitespace
-s = 1*( SP / HTAB / CR / LF / %x3000 )
+; Whitespace, optionally prepended with LRM
+s = [%x200E] 1*( SP / HTAB / CR / LF / %x3000 )
```

Comment:
This means that LRM is _not_ valid after a line terminator, e.g. before the quoted patterns in a message like
```
.match {$count :number}
one
	{{You have {$count} notification.}}
*
	{{You have {$count} notifications.}}
```

Is that intentional?

Reply:
No, not really.

Probably what needs to happen here is a distinguishing of optional and non-optional whitespace. Everywhere we have `[s]` should use a production that can be just LRM (or RLM, fwiw) or nothing, e.g.:

```abnf
; optional whitespace
owsp = *( SP/ HTAB / CR / LF / %x3000 / %x200E / %x200F )
```

And everywhere that requires positive whitespace (i.e. just `s`) permit controls to either side:

```abnf
; required whitespace
wsp = [ (%x200E / %x200F) ] 1*( SP / HTAB / CR / LF / %x3000) [ (%x200E / %x200F) ]
```

Reply:
I think that would be an improvement (and then you can drop the convoluted explanation from the conformance statement, and the profile limits itself to changing the sets of characters).

Reply:
I did the hard change. We now need to get WG approval.